ci/acceptance-test/multibox/10.0.100.14-vdc-executor-lxc/guestroot/etc/openvdc/executor.toml

```diff
@@ -2,4 +2,4 @@
 driver = "lxc"
 
 [zookeeper]
-endpoint = "zk://localhost/openvdc"
+endpoint = "zk://10.0.100.10/openvdc"
```

Comment:
My experience was that this line gets ignored which is why I didn't fill it in. I'd like to discuss a bit what exactly is going on here before merge.